modules/core/src/main/scala/doobie/hi/connection.scala

```diff
@@ -226,6 +248,18 @@ object connection {
     } yield ele
   }
 
+  def stream[A: Read](
+      prepared: PreparedExecutionStream,
+      loggingInfo: LoggingInfo
```

Comment:
I'm not sure these methods are really needed? Especially since the current API does not really allow users to work with (execute) a PreparedExecutionStream directly